homeassistant/components/climate/mill.py

```diff
@@ -114,13 +114,16 @@ def device_state_attributes(self):
         """Return the state attributes."""
         if self._heater.room:
             room = self._heater.room.name
+            avg_room_temp = self._heater.room.avg_temp
```

Comment:
shouldn't this be the value returned from `current_temperature` ?

Reply:
No, you can have several heaters in one room (Defined by Mill app). This will be the average temperature for in that room.